presto-main/src/test/java/io/prestosql/server/security/oauth2/TestOAuth2WebUiAuthenticationFilter.java

```diff
@@ -198,15 +199,17 @@ public void testSuccessfulFlow()
     }
 
     @Test
+    @Flaky(issue = "https://github.com/prestosql/presto/issues/6223", match = "Presto-OAuth2-Token is missing")
```

Comment:
Can you please explain in the commit message why it is difficult to provide actual fix for that?